README.md

```diff
@@ -22,3 +22,9 @@ The project supports reading stream metrics from the following services:
 ## Contributing
 
 This application was built with ASP.NET Core 2.0 and can be built on Mac, Linux, and Windows.  Download the [.NET SDK](https://dot.net) and grab a copy of [Visual Studio Code](https://code.visualstudio.com) to get started on any platform
+
+### Naming guideline for unit tests:
```

Comment:
@csharpfritz, I think I initially branched off of master instead of dev and unintentionally picked up this commit